io/js/src/main/scala/fs2/io/internal/facade/os.scala

```diff
@@ -42,6 +42,10 @@ package object os {
   @JSImport("os", "networkInterfaces")
   private[io] def networkInterfaces(): js.Dictionary[js.Array[NetworkInterfaceInfo]] = js.native
 
+  @js.native
+  @JSImport("os", "EOL")
+  private[io] def EOL(): String = js.native
```

Comment:
No parens, since that's now how it's defined in Node.js :)

https://nodejs.org/api/os.html#oseol

```suggestion
  private[io] def EOL: String = js.native
```